lib/softlayer/Config.rb

```diff
@@ -95,7 +95,12 @@ def Config.file_settings(*additional_files)
 						result[:username] = softlayer_section['username'] if softlayer_section['username']
 						result[:endpoint_url] = softlayer_section['endpoint_url'] if softlayer_section['endpoint_url']
 						result[:api_key] = softlayer_section['api_key'] if softlayer_section['api_key']
-						result[:timeout] = softlayer_section['timeout'] if softlayer_section['timeout']
+
+            begin
```

Comment:
Hazards of switching editors without paying close attention to the "insert tabs as spaces" setting